### DISCUSSION TITLE
Provide a "fat" variant of gitHub actions default container image

### DISCUSSION THREAD

**Discussion by ambravo** [engswee/flashpipe#39] [Ideas]
Would it be possible to provide a "fat" variant of the default image that includes common tools like curl?

Currently having to apt-get install curl in many workflows which is inefficient. A fat image variant would save time and simplify workflows while keeping the default image slim.

https://github.com/engswee/flashpipe/blob/687862ea650f903531ae0dfe13d1459582c95847/.github/workflows/go-prod.yml#L77

**Comment by engswee** [comment]
Could you share the use case of having curl, etc in the image? Would love to hear how curl is used in combination with flashpipe in workflows, so that I can see if it would be a worthwhile addition.

PS: Are you back in the SAP ecosystem after being away for a while? 😉

**Comment by engswee** [comment]
Coming back to this after a long while and I think with the versatility of Docker images, this does not need to be handled within the `flashpipe` solution.

Users can build their own Docker images using the flashpipe image as the base, and then install any further tools that are needed. They can they use their own image in the workflows.
